### PR TITLE
Style sign in

### PR DESCRIPTION
**Pull request by mingyokim**
## :construction_worker: Changes

This PR styles the sign in form! (and only sign in form - password recovery or apply now link don't actually point to anything)

login_success

login_fail

## :thought_balloon: Notes

This PR is based on top of #22 - do not merge until that's merged first!!

This PR introduces `PasswordInput` component - may make that into a separate PR to make this PR smaller

## :flashlight: Testing Instructions

Try signing in! 

E: test@example.com
Pw: testtest

### CODE REVIEW

File: web/components/app/App.sass [hunk 1]
```diff
@@ -1,17 +1,8 @@
-.App
-  text-align: center
+#app
```
Comment (bobheadxi) [inline web/components/app/App.sass]: never seen this syntax before, but if it works it works

File: web/components/auth/Login/Login.sass [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            color: $primary
+
+            > a
+              &:visited
+                color: $primary
+
+              &:hover
+                color: $secondary-highlight
+
+        > div.card-right
```
Comment (bobheadxi) [inline web/components/auth/Login/Login.sass]: stylesheets looking clean ✨

File: web/components/input/text/TextInput/TextInput.sass [hunk 3]
```diff
@@ -52,5 +50,6 @@ div.text-input
 
   p
     margin-top: 4px
+    margin-bottom: -18px
```
Comment (bobheadxi) [inline web/components/input/text/TextInput/TextInput.sass]: this seems unfortunately hacky 🤔 what caused this to be needed?

Comment (mingyokim) [reply web/components/input/text/TextInput/TextInput.sass]: i forgot lol

**Pull request merged**